### PR TITLE
Update the 1.x branch for new proto files and to keep the conformance tests building

### PR DESCRIPTION
**Pull request by thomasvl**

### CODE REVIEW

**Review by allevato — APPROVED** [review]

**Comment by thomasvl** [comment]
Comparing the "Update and install dependencies" step, it seems with Swift 4.2.4 images doesn't check _bionic-updates_ to get the newest cmake.

@Lukasa @tbkka ideas how to tweak that?  Is there a `add-apt-repository` that in like we do for git on the branch?

**Comment by Lukasa** [comment]
I believe kitware host a PPA that could be used for this purpose: https://apt.kitware.com.

**Comment by thomasvl** [comment]
https://github.com/protocolbuffers/protobuf/pull/10751 might end up fixing the issue, will retrigger this after that lands.

**Comment by thomasvl** [comment]
> [protocolbuffers/protobuf#10751](https://github.com/protocolbuffers/protobuf/pull/10751) might end up fixing the issue, will retrigger this after that lands.

So this didn't seem to help.

> I believe kitware host a PPA that could be used for this purpose: https://apt.kitware.com.

Showing my linux distro ignorance, what out of that long list do I need to add?

**Comment by thomasvl** [comment]
@Lukasa I think I've got it, can you take a look at the new commit to make sure things LG?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -74,8 +83,19 @@ jobs:
...
-    - name: Install newer git for bionic
-      # bionic's git doesn't do modules via the REST api, update the git version.
+    - name: Add Kitware Repository
+      # xenial cmake is to old, update it.
+      if: matrix.update_cmake
+      run: |
+        set -eu
+        apt-get install -y apt-transport-https software-properties-common
+        curl -s https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | apt-key add -
+        apt-add-repository 'deb https://apt.kitware.com/ubuntu/ xenial main'
```
[the remark this answers is not in the document]
Comment (thomasvl) [reply .github/workflows/build.yml]: Thanks, looks like it's working.

**Pull request merged**